### DISCUSSION TITLE
Is there any way to integrate (wrap) bootstrap or any JS lib inside to create rust components

### DISCUSSION THREAD

**Discussion by antonio-castellon** [DioxusLabs/dioxus#1980] [Q&A]
This way to work with RUST is amazing but I feel (maybe wrongly) that we need some additional components in order to not waste our time re-coding again ourselves some complex components like tables with filters, sort, events, etc. like we are doing in react. If is there a way to include inside these Rust components some library (or way to do that) in order to re-use all these code, will be fantastic... in any other case we must need to wait until the ecosystem has enough complex components to simplify the full code cycle in cross systems.
:-)

**Comment by ealmloff** [comment]
For tables, you might be interested in https://github.com/feral-dot-io/dioxus-sortable?tab=readme-ov-file

Integrating with other frameworks components is discussed more in these comments:
- https://github.com/DioxusLabs/dioxus/discussions/251#discussioncomment-2186669
- https://github.com/DioxusLabs/dioxus/discussions/1506#discussioncomment-7151987
